hammer/config/defaults.yml

```diff
@@ -92,6 +92,8 @@ vlsi.technology:
   # the extracted contents of foobar.tar.gz.
   # If this is not specified, then the tarballs will be extracted to obj/<tech_dir>/extracted/.
 
+  timing_lib_pref: "NLDM"
```

Comment:
Can you add some comments below to describe what this key does and the available options?

Reply:
Also, this key isn't used in `get_timing_lib_with_preference`?

Reply:
Re 1: Added
Re 2:  It accepts a preference argument and defaults to NLDM if none is provided.